src/main/java/net/atos/zac/app/zaken/model/RESTGerelateerdeZaak.java

```diff
@@ -9,26 +9,13 @@
 
 public class RESTGerelateerdeZaak {
 
-    public enum RelatieType {
-        DEELZAAK,
-        HOOFDZAAK,
-        VERVOLG,
-        RELEVANT,
-        BIJDRAGE
-    }
-
     public RelatieType relatieType;
 
-    public String identificatie;
+    public String zaaknummer;
 
-    public String omschrijving;
+    public String zaaktype;
```

Comment:
noem deze zaaktypeIdentificatie. En weet je zeker dat je niet eigenlijk zaaktypeOmschrijving moet hebben? Want op de meeste plaatsen gebruiken we (dacht ik) de zaaktypeOmschrijving om te tonen. Maar weet dat niet zeker